bolt-project.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Are module dependencies going to be managed in `bolt-project.yaml` or in `Puppetfile`?

Reply:
Right now I am keeping both, but lets have talk around it.